cmd/statusd/debug/commands.go

```diff
@@ -0,0 +1,175 @@
+			case token.CHAR:
+				args[i] = expr.Value[1]
+			case token.STRING:
+				r := strings.NewReplacer("\\n", "\n", "\\t", "\t", "\\\"", "\"")
+				args[i] = strings.Trim(r.Replace(expr.Value), `"`)
+			}
+		case *ast.Ident:
+			switch expr.Name {
+			case "true":
+				args[i] = true
+			case "false":
+				args[i] = false
+			default:
+				args[i] = expr.Name
+			}
+		default:
+			args[i] = fmt.Sprintf("[unknown: %#v]", expr)
+		}
+	}
+	return args
+}
+
+// commandSet implements the set of commands the debugger unterstands.
+// In the beginning a subset of the Status API, may later grow to
+// utility commands.
+//
+// Direct invocation of commands on the Status API sometimes sadly
+// is not possible due to non-low-level arguments. Here this wrapper
+// helps.
+type commandSet struct {
+	statusAPI *api.StatusAPI
+}
+
+// newCommandSet creates the command set for the passed Status API
+// instance.
+func newCommandSet(statusAPI *api.StatusAPI) *commandSet {
+	return &commandSet{
+		statusAPI: statusAPI,
+	}
+}
+
+// StartNode loads the configuration out of the passed string and
+// starts a node with it.
+func (cs *commandSet) StartNode(config string) error {
+	nodeConfig, err := params.LoadNodeConfig(config)
+	if err != nil {
+		return err
+	}
+	return cs.statusAPI.StartNode(nodeConfig)
+}
+
+// StopNode starts the stopped node.
+func (cs *commandSet) StopNode() error {
+	return cs.statusAPI.StopNode()
+}
+
+// ResetChainData removes chain data from data directory.
+func (cs *commandSet) ResetChainData() error {
+	_, err := cs.statusAPI.ResetChainDataAsync()
+	return err
+}
+
+// CallRPC calls status node via RPC.
+func (cs *commandSet) CallRPC(inputJSON string) string {
+	return cs.statusAPI.CallRPC(inputJSON)
+}
+
+// CreateAccount creates an internal geth account.
+func (cs *commandSet) CreateAccount(password string) (string, string, string, error) {
+	return cs.statusAPI.CreateAccount(password)
+}
+
+// CreateChildAccount creates a sub-account.
+func (cs *commandSet) CreateChildAccount(parentAddress, password string) (string, string, error) {
+	return cs.statusAPI.CreateChildAccount(parentAddress, password)
+}
+
+// RecoverAccount re-creates the master key using the given details.
+func (cs *commandSet) RecoverAccount(password, mnemonic string) (string, string, error) {
+	return cs.statusAPI.RecoverAccount(password, mnemonic)
+}
+
+// SelectAccount selects the addressed account.
+func (cs *commandSet) SelectAccount(address, password string) error {
+	return cs.statusAPI.SelectAccount(address, password)
+}
+
+// Logout clears the Whisper identities.
+func (cs *commandSet) Logout() error {
+	return cs.statusAPI.Logout()
+}
+
+// CompleteTransaction instructs API to complete sending of a given transaction.
+func (cs *commandSet) CompleteTransaction(id, password string) (string, error) {
+	txHash, err := cs.statusAPI.CompleteTransaction(common.QueuedTxID(id), password)
+	if err != nil {
+		return "", err
+	}
+	return txHash.Hex(), nil
```

Comment:
I would use `txHash.String()` here — currently it calls `.Hex()` underneath, but if representation format is changed (as it happened with EIP55, for example), we would not need to update the code or worry about string result correctness.

Reply:
Done.